cmd/gateway/commands.go

```diff
@@ -118,12 +120,17 @@ func createStaticModeCommand() *cobra.Command {
 	// flag values
 	gateway := namespacedNameValue{}
 	var updateGCStatus bool
+	configName := stringValidatingValue{
```

Comment:
can we add some testing for this flag in TestStaticModeCmdFlagValidation in commands_test.go ?